From: John Johansen <john.johansen@canonical.com>
Date: Mon, 29 Jan 2018 12:26:43 +0000
Subject: When specifying '-O' to the parser, it skips reading the cache.

On Ubuntu, due to LP: #1383858 we want to use no-expr-simplify, but also use
the cache if it is there.
Forwarded: no (Ubuntu-specific)
Bug: https://launchpad.net/bugs/1385947
---
 parser/parser_main.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/parser/parser_main.c b/parser/parser_main.c
index 80c243d..35bbd30 100644
--- a/parser/parser_main.c
+++ b/parser/parser_main.c
@@ -439,8 +439,6 @@ static int process_arg(int c, char *optarg)
 		}
 		break;
 	case 'O':
-		skip_read_cache = 1;
-
 		if (!handle_flag_table(optflag_table, optarg,
 				       &dfaflags)) {
 			PERROR("%s: Invalid --Optimize option %s\n",
